raiden/transfer/mediated_transfer/mediator.py

```diff
@@ -128,6 +137,25 @@ def is_send_transfer_almost_equal(
     )
 
 
+def filter_reachable_routes(
+        transfers_pair: typing.List[MediationPairState],
+        routes: typing.List[RouteState],
+        nodeaddresses_to_networkstates: typing.NodeNetworkStateMap,
+):
+    """This function makes sure we use reachable routes only."""
+    address_to_route = {r.node_address: r for r in routes}
+
+    for pair in transfers_pair:
```

Comment:
This is not right, it should iterate over `routes` here, and not  `transfers_pair`